### PR TITLE
refactor: Make syntax friendlier with getColor and getVariable

### PR DESCRIPTION
**Pull request by misund**

### CODE REVIEW

File: src/atoms/DymoHeading.js [hunk 1]
```diff
@@ -15,16 +16,16 @@ const PaddedText = styled.span`
...
 		;
-		background: ${props => props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]};
-		color:      ${props => props.theme.colors[props.theme.colors.skinColors[props.skin.textColor]]};
+		background: ${props => getColor(props.skin.backgroundColor)};
+		color:      ${props => getColor(props.skin.textColor)};
 		-webkit-box-decoration-break: clone;
 
 		span.highlighted {
-			color: ${props => props.theme.colors.red};
+			color: ${getColor('red')};
```
Comment (SindreSvendby) [inline src/atoms/DymoHeading.js]: this seems fine! I think!

File: src/atoms/DymoHeading.js [hunk 2]
```diff
@@ -15,16 +16,16 @@ const PaddedText = styled.span`
...
-			calc(1/2*${props => props.theme.variables.horizontalBase})
+			calc(1/2 * ${getVariable('horizontalBase')})
 			0
-		  ${props           => props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]}
+		  ${props => getColor(props.skin.backgroundColor)}
 		;
-		background: ${props => props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]};
-		color:      ${props => props.theme.colors[props.theme.colors.skinColors[props.skin.textColor]]};
+		background: ${props => getColor(props.skin.backgroundColor)};
+		color:      ${props => getColor(props.skin.textColor)};
```
Comment (SindreSvendby) [inline src/atoms/DymoHeading.js]: why do we do `${props => getColor(props.skin.textColor)};` is in not possible to do 
`${getColor(props.skin.textColor)}`;

if all the variables is uniq it is possible to also create const foreach so we get `${getColor(TEXT_COLOR)}`; or similar, but here we should include the designers!

Comment (misund) [reply src/atoms/DymoHeading.js]: `getColor` expects a string that is defined either on the `skinColors` object or directly in the color palette. If we hard code the color, `${getColor('red')}` is fine. In this case, though, the color name is dynamically fetched from a prop. To access the props, we still need the `${props => props.myProp}` syntax.

Comment (misund) [reply src/atoms/DymoHeading.js]: However, I'd say that still is a significant improvement from
```
${props => (
  props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]
  || props.theme.colors[props.skin.backgroundColor]}
);
```

**Pull request merged**